scout/server/blueprints/variant/templates/variant/utils.html

```diff
@@ -316,16 +316,20 @@ <h6>Phenotype terms (HPO):</h6>
   {% set align_start = variant.position - 500 if variant.position > 500 else 0 %}
   {% set align_end = variant.end + 500 %}
   {% set chrom = variant.chromosome %}
-    <div class="col-4 offset-1">
+    <div class="col-4 offset-1 ">
     {% if zoom == "start" %}
       {% set align_end = variant.position + 500 %}
-      {{variant.chromosome}}:{{variant.position}}
+      {{variant.chromosome}}:{{variant.position|human_longint|safe}}<button type="button" class="fa fa-file-text btn-xs js-tooltip js-copy"
```

Comment:
Preserve the possibility for hassle-free copying of numbers (without special chars interleaved in them).